solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseCloudSolrClient.java

```diff
@@ -1052,10 +1074,12 @@ public RouteException(ErrorCode errorCode, NamedList<Throwable> throwables, Map<
       }
     }
 
-    return resp;
+    return respFuture;
```

Comment:
very minor:  I'd prefer "rspFuture".  I see "response" abbreviated to "rsp" in general, much less to "resp".